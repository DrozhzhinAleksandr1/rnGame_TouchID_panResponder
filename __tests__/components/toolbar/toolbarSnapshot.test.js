import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { ToolBar } from '/components/toolbar/toolbar';

it('create snapshot toolbar', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<ToolBar />);
    expect(toolbarSnapshot).toMatchSnapshot();
});
it('create snapshot toolbar', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<ToolBar showSearch={false} />);
    expect(toolbarSnapshot).toMatchSnapshot();
});