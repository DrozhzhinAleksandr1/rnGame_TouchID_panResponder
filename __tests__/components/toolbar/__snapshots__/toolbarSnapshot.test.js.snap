// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot toolbar 1`] = `
<Component
  style={
    Object {
      "backgroundColor": "#172533",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "space-between",
      "maxHeight": 56,
      "minHeight": 56,
      "shadowColor": "rgba(0, 0, 0, 0.15)",
      "shadowOffset": Object {
        "height": 4,
        "width": 0,
      },
      "shadowOpacity": 1,
      "shadowRadius": 4,
      "zIndex": 1,
    }
  }
>
  <Component
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
      }
    }
  >
    <withNavigation(Connect(Menu)) />
    <Connect(Title) />
    <Connect(Search) />
  </Component>
</Component>
`;

exports[`create snapshot toolbar 2`] = `
<Component
  style={
    Object {
      "backgroundColor": "#172533",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "space-between",
      "maxHeight": 56,
      "minHeight": 56,
      "shadowColor": "rgba(0, 0, 0, 0.15)",
      "shadowOffset": Object {
        "height": 4,
        "width": 0,
      },
      "shadowOpacity": 1,
      "shadowRadius": 4,
      "zIndex": 1,
    }
  }
>
  <Component
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
      }
    }
  >
    <withNavigation(Connect(Menu)) />
    <Connect(Title) />
    <Connect(Search) />
  </Component>
</Component>
`;
